### PR TITLE
Update infinitybotlist.com to be infinitybots.gg

### PR DESCRIPTION
**Pull request by TheRealToxicDev**

### CODE REVIEW

**Comment by TheRealToxicDev** [comment]
I removed all old legacy data hopefully this change won’t reflect negatively on the site. However if I need to readd them I can 👌🏻

File: data/legacy.json [hunk 1]
```diff
@@ -3,8 +3,7 @@
     "vultrex.io": "discordbots.co",
     "space-bot-list.org": "space-bot-list.xyz",
     "mythicalbots.xyz": "bots.idledev.org",
-    "infinitybots.xyz": "infinitybotlist.com",
-    "infinitybots.com": "infinitybotlist.com",
```
Comment (MattIPv4) [inline data/legacy.json:7]: You need to keep these existing legacy items but change the values to be the new ID -- removing them will break users that are currently using these old IDs still

File: data/legacy.json [hunk 2]
```diff
@@ -3,8 +3,7 @@
...
+    "infinitybots.gg": "infinitybots.gg",
```
Comment (MattIPv4) [inline data/legacy.json:6]: A legacy ID cannot map to itself, that makes no sense. This should make the old ID to the new ID (`nfinitybotlist.com` -> `infinitybots.gg`)

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: data/legacy.json (hunk 2 above)
Comment (TheRealToxicDev) [reply data/legacy.json]: Re-added and updated

File: data/legacy.json [hunk 3]
```diff
@@ -3,8 +3,9 @@
     "vultrex.io": "discordbots.co",
     "space-bot-list.org": "space-bot-list.xyz",
     "mythicalbots.xyz": "bots.idledev.org",
-    "infinitybots.xyz": "infinitybotlist.com",
-    "infinitybots.com": "infinitybotlist.com",
+    "infinitybots.xyz": "infinitybots.gg",
+    "infinitybots.com": "infinitybots.gg",
+    "nfinitybots.com": "infinitybots.gg",
```
Comment (MattIPv4) [inline data/legacy.json:8]:
```suggestion
    "infinitybotlist.com": "infinitybots.gg",
```

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: data/legacy.json (hunk 3 above)
Comment (TheRealToxicDev) [reply data/legacy.json]: Suggested change has been committed

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**